lib/matplotlib/tests/test_ticker.py

```diff
@@ -521,6 +525,13 @@ def test_sublabel(self):
         ax.set_xlim(0.5, 0.9)
         self._sub_labels(ax.xaxis, subs=np.arange(2, 10, dtype=int))
 
+    @pytest.mark.parametrize('val', call_data)
+    def test_LogFormatter_call(self, val):
+        # test _num_to_string method used in __call__
+        temp_lf = mticker.LogFormatter()
+        temp_lf.axis = FakeAxis()
+        assert str(temp_lf(val)) is not None
```

Comment:
this is still testing that result `is not None` (and `str(None) is not None`).  Can you check that these are equal to the expected value?